### ISSUE TITLE
Create a logo for Tech Tutor application

### ISSUE THREAD

**Issue by Vaishnavimore10** [Google-Developer-Student-Club-CCOEW/Flutter#7]
Create a logo for Tech Tutor application
It should be according to given colour scheme:
https://colorhunt.co/palette/00abb33c4048b2b2b2eaeaea

You can upload your design here:
https://drive.google.com/drive/folders/1FGu2cDMS-UmKbEJ9hAvyyomZxHfd86LH?usp=sharing

**Comment by disha100** [comment]
I want to work on this ..please assign

**Comment by Vaishnavimore10** [comment]
ok.  You can work on this issue

**Comment by disha100** [comment]
how do i create a pr for this issue ...

**Comment by Vaishnavimore10** [comment]
This issue has been assigned to you . You can upload your logo design in the drive link provided and create a new pull request to this issue. once the design is approved . I will merge your pr. Ask if have any doubts.

**Comment by disha100** [comment]
OK

**Comment by disha100** [comment]
HEY! @Vaishnavimore10 
I DON'T HAVE ACCESS TO EDIT THIS FOLDER ..
PLEASE CHECK

**Comment by Vaishnavimore10** [comment]
I think you should be able to . The drive has access to edit for anyone with link. Please try again

**Comment by disha100** [comment]
done ! 
please review
